cristin-project/src/test/java/no/unit/nva/cristin/projects/NvaProjectBuilderTest.java

```diff
@@ -90,4 +97,14 @@ void shouldSerializeAndDeserializeFundingAmountIntoSameObject() {
         assertThat(deserialized, equalTo(fundingAmount));
     }
 
+    @Test
+    void shouldSerializeAndDeserializeTypedLabelIntoSameObject() {
```

Comment:
Remove these tests and keep a single top-level (i.e. the root class for each data structure) that ensures that the entire tree can be built (it will cover all these cases too)

Reply:
Will do in separate PR. It requires some work on the model classes